docs/pages/kubernetes-access/guides/discovery.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Kubernetes Discovery Guides (Preview)
```

Comment:
```suggestion
title: Kubernetes Discovery (Preview)
```
Suggesting naming the page after the feature, rather than the fact that it contains guides.